### PR TITLE
DF-2167 - current openings page

### PR DESCRIPTION
**Pull request by anselmbradford**
Adds current openings page to careers.

## Additions

- Adds `/careers/current-openings/`.

## Changes

- Moves content formatting macros from `macros.html` to `/macros/util/format.html`.
- Converted excerpts to articles from sections in the careers section.

## Testing

- `/careers/current-openings/`.

## Review

- @KimberlyMunoz 
- @sebworks 
- @jimmynotjim 

## Preview

screen shot 2015-07-02 at 5 08 29 pm

screen shot 2015-07-02 at 5 08 35 pm

## Notes

- Added mock data for the jobs table. Also these will need to be converted to links to the individual job detail pages in another task/PR.

### CODE REVIEW

**Comment by sebworks** [comment]
Did you mean to create `contact-us/general-inqueries.html` ?

**Comment by sebworks** [comment]
Can we create a generic macro where you would be able to pass in row and table keys?

**Comment by KimberlyMunoz** [comment]
The responsive version of this table is a little wonky. Adding `simple-table` to the table should format it to something a little more useful.

image

**Comment by anselmbradford** [comment]
> Did you mean to create contact-us/general-inqueries.html ?

Haha, nope. Fixed in 
https://github.com/cfpb/cfgov-refresh/commit/a010b693aa3c9d0cf952073d4b91e254c87ccd59

**Comment by anselmbradford** [comment]
> The responsive version of this table is a little wonky. Adding simple-table to the table should format it to something a little more useful.

Added in https://github.com/cfpb/cfgov-refresh/commit/e377e8bca51218872c5c4fdf770cfc112790d1a0

**Comment by anselmbradford** [comment]
> Can we create a generic table macro where you would be able to pass in row and table keys?

This sounds good to me! Would it be okay to do this in another PR? We don't know yet what the structure of the job list data will be. Opened https://github.com/cfpb/cfgov-refresh/issues/712 for this.

**Comment by sebworks** [comment]
Yep, that works. :+1:

**Comment by KimberlyMunoz** [comment]
:+1:

**Comment by anselmbradford** [comment]
Removes table column width classes per @jimmynotjim in-person comments in https://github.com/cfpb/cfgov-refresh/commit/6670dab7d68ff83936b95a2bf4bb3476d240453c

Table now lays out better:
screen shot 2015-07-06 at 11 04 21 am

File: careers/current-openings/index.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+{% block content_main %}
+
+    {% import "_macro-table-current-openings.html" as table_current_openings %}
+
+    <section class="block
+                    block__flush-top
+                    block__sub">
+            <h1>Current Openings</h1>
+            <p class="h3">
+                Find the position that’s right for you. See positions by title, location, or closing date. We recommend applying at least a full day before the closing date, in case you have issues or questions you need answered.
```
Comment (jimmynotjim) [inline careers/current-openings/index.html]: Add a semantic line break

Comment (anselmbradford) [inline careers/current-openings/index.html]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/0081829f181bd6df229cb4d3f34c217665f5306d

**Comment by anselmbradford** [comment]
Adds mobile view labels in https://github.com/cfpb/cfgov-refresh/commit/195a1d449e63393bf6ac58060b0cdb1fb454afd1 screen shot 2015-07-06 at 12 50 49 pm

**Comment by anselmbradford** [comment]
Mergin' w/ @jimmynotjim's verbal approval

**Pull request merged**